src/app/parcels/parcelsTable/sortableColumns.ts

```diff
@@ -36,6 +44,7 @@ const parcelsSortableColumns: SortOptions<ParcelsTableRow, ParcelsSortMethod>[]
         key: "packingDate",
         sortMethod: (query, sortDirection) =>
             query
+                .order("client_is_active")
```

Comment:
Does this need to be `ascending: sortDirection !== "asc"`? From what I understand all these just push inactive clients to the end?

Reply:
Thanks for catching this - I've fixed this so that, when sorting by this field or the default sorting, the sorting is now

Date (ascending or descending depending on how many times the header has been clicked)
Packing Shift (AM/PM/Shift1,2,3,4)
Method (Delivery, then Collection)
Client inactivity (active clients, then inactive clients)
Postcode
Collection centre

I think this is the expected / desired behaviour

Reply:
I think that's correct - and it covers accidentally revealing deleted postcodes, right? Re #206

Reply:
yes, because we sort by the client inactivity first, then the sensitive info field (postcode), the postcode is never revealed